shared/testutil/require/requires.go

```diff
@@ -0,0 +1,30 @@
+package assert
```

Comment:
Same here, `testutil.RequireEqual` instead of creating a separate package for it.

Reply:
Also the package name is incorrect here

Reply:
Thanks, package name was wrong indeed, fixed.